zuul.d/nodesets.yaml

```diff
@@ -942,20 +942,20 @@
           - ubuntu-bionic
 
 - nodeset:
-    name: Trendmicro-DeepSecurity-20.0.198-python38
+    name: Trendmicro-DeepSecurity-20.0.344-python38
     nodes:
       - name: ubuntu-bionic
         label: ubuntu-bionic-1vcpu
-      - name: Trendmicro-DeepSecurity-20.0.198
-        label: Trendmicro-DeepSecurity-20.0.198
+      - name: Trendmicro-DeepSecurity-20.0.344
+        label: Trendmicro-DeepSecurity-20.0.344
     groups:
       - name: appliance-ssh
         nodes:
-          - Trendmicro-DeepSecurity-20.0.198
+          - Trendmicro-DeepSecurity-20.0.344
```

Comment:
You ask for `Trendmicro-DeepSecurity-20.0.344`, but I only see `Trendmicro-DeepSecurity-12.5.855` in https://github.com/ansible-network/windmill-config/blob/master/nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml#L13

Reply:
protip : You can get a list of the available labels here: https://dashboard.zuul.ansible.com/t/ansible/labels